Test group_test.spec.ts Duplicate group fails repeatedly

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

testsuite

### Describe the bug

The test group_test.spec.ts Duplicate group repeatedly for example in this run:

https://github.com/keycloak/keycloak/actions/runs/11124470293/job/30914009482?pr=33422

```
  Running:  group_test.spec.ts                                                              (2 of 7)

  Group test
    List
      ✓ Create group from empty option (7188ms)
      ✓ Create group from search bar (4534ms)
      ✓ Fail to create group with empty name (2054ms)
      ✓ Fail to create group with duplicated name (4088ms)
      ✓ Empty search (1784ms)
      ✓ Search group that exists (2585ms)
      ✓ Search group that does not exists (2134ms)
      (Attempt 1 of 4) Duplicate group
      (Attempt 2 of 4) Duplicate group
      (Attempt 3 of 4) Duplicate group
      ✓ Delete group from item bar (5506ms)
      ...
  43 passing (4m)
  1 pending
  1 failing
  1) Group test
       List
         Duplicate group:
     AssertionError: Timed out retrying after 30000ms: Unable to find an element by: [data-testid="last-alert"]
```

We are going to skip the test for now.

### Version

999

### Regression

- [ ] The issue is a regression

### Expected behavior

The test should run OK.

### Actual behavior

Test is failing.

### How to Reproduce?

```bash
CYPRESS_BASE_URL=http://localhost:8080/admin/ CYPRESS_KEYCLOAK_SERVER=http://localhost:8080 $PNPM_HOME/pnpm run cy:run --browser chromium --spec ./cypress/e2e/group_test.spec.ts
```

### Anything else?

_No response_

--- a/js/apps/admin-ui/cypress/support/util/ModalUtils.ts
+++ b/js/apps/admin-ui/cypress/support/util/ModalUtils.ts
@@ -31,7 +31,7 @@ export default class ModalUtils extends PageObject {
   }
 
   confirmDuplicateModal() {
-    cy.findByTestId(this.#confirmDuplicateModalBtn).click({ force: true });
+    cy.findByTestId(this.#confirmDuplicateModalBtn).click();
 
     return this;
   }

